.editorconfig

```diff
@@ -0,0 +1,25 @@
+# top-most EditorConfig file
+root = true
+
+# Apply to all files
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# Kotlin specific rules
+[*.{kt,kts}]
+max_line_length=120
+
+# Ruby specific rules
+[*.rb]
+indent_style = space
+indent_size = 2
+
+# Fastlane specific rules
+[**Fastfile]
+indent_style = space
+
+# YML specific fules
+[*.yml]
+trim_trailing_whitespace=true
```

Comment:
Isn't that already provided by the rules in `[*]`?

Reply:
Oh, right! Fixed :-)